recommenders/models/rlrmc/RLRMCdataset.py

```diff
@@ -68,8 +68,7 @@ def _data_processing(self, train, validation=None, test=None, mean_center=True):
         """
         # Data processing and reindexing code is adopted from https://github.com/Microsoft/Recommenders/blob/main/recommenders/models/ncf/dataset.py
         # If validation dataset is None
-        df = train if validation is None else pd.concat([train, validation])
-        df = df if test is None else pd.concat([df, test])
+        df = pd.concat(filter(lambda x: x is not None, [train, validation, test]))
```

Comment:
`concat` handles `None` nicely, so that you can simply:
`df = pd.concat([train, test, validation])`

Reply:
> Any None objects will be dropped silently unless they are all None in which case a ValueError will be raised.
It's true! Train won't ever be none so it will never fail.